### PR TITLE
some small improvements and corrections

### PR DESCRIPTION
**Pull request by PascalSchumacher**

### CODE REVIEW

File: templates/assertj-core-features-highlight-template.html [hunk 1]
```diff
@@ -15,7 +15,7 @@
    <div class="col-lg-10 col-md-10 col-sm-10 text-left" >
       <h1 class="page-header">AssertJ Core features highlight</h1>
 
-      <p>AssertJ has some great features that people don't always know, the purpose of this page is to show them so that you take the most of AssertJ.</p>
+      <p>AssertJ has many great features that not everbody is aware of. The purpose of this page is to show them to you, so that you will get the most out of AssertJ.</p>
```
Comment (joel-costigliola) [inline templates/assertj-core-features-highlight-template.html]: small correction on small corrections : `everbody`  :)

Comment (PascalSchumacher) [inline templates/assertj-core-features-highlight-template.html]: got me :)

**Comment by PascalSchumacher** [comment]
rebased onto current master

**Comment by joel-costigliola** [comment]
Sweet thanks.

**Pull request merged**